stl/inc/type_traits

```diff
@@ -1151,11 +1151,20 @@ template <size_t _Len, class... _Types>
 using aligned_union_t = typename aligned_union<_Len, _Types...>::type;
 
 // STRUCT TEMPLATE underlying_type
+template <class _Ty, bool = is_enum_v<_Ty>>
+struct _Underlying_type {};
+
+template <class _Ty>

```

Comment:
I dont think that this specialization is needed. It doesnt differ from the definition above.

Reply:
Good point.